### ISSUE TITLE
Forecast IO bug?

### ISSUE THREAD

**Issue by MarcelErdmann** [xibosignage/xibo#238]
Hello,

I miss the color field . After the paste of weather .
Only after saving and editing the color box appears .

Here are 2 pictures

---------------------------------
New Forecast
---------------------------------
![forecast1](https://cloud.githubusercontent.com/assets/10828284/6167790/4ac4c460-b2bf-11e4-84dd-928ed3fee894.jpg)

---------------------------------
Edit Forecast
---------------------------------
![forecast2](https://cloud.githubusercontent.com/assets/10828284/6167809/8df35be8-b2bf-11e4-8daf-9d3d2556576e.jpg)

**Comment by dasgarner** [comment]
You are right, it's missing from the Add Form - tagged as a bug.

**Comment by dasgarner** [comment]
Tracked in new issue
